src/Features/Core/Portable/ImplementAbstractClass/AbstractImplementAbstractClassService.State.cs

```diff
@@ -56,7 +56,7 @@ private class State
                     return null;
                 }
 
-                var unimplementedMembers = classType.GetAllUnimplementedMembers(
+                var unimplementedMembers = classType.GetAllUnimplementedMembersNotRequiringExplicitImplementation(
                     SpecializedCollections.SingletonEnumerable(abstractClassType), cancellationToken);
```

Comment:
i don't love this.  would prefer this as an option to `GetAllUnimplementedMembers`